### PR TITLE
[KR] fixes and translated BBCode

### PR DESCRIPTION
**Pull request by EverTokki**
### Description
Some Korean translation fixes and additions.

### Status
Done

### Changelog
- About - fixed awkward sentencing
- BanchoBot - fixed typo
- BBCode - translated article into Korean

### CODE REVIEW

File: wiki/BBCode/ko.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
+# BBCode
```
Comment (MegaApplePi) [inline wiki/BBCode/ko.md]: Can you put the language name after this article's title?
`# BBCode` -> `# BBCode (한국어)`

Comment (MegaApplePi) [reply wiki/BBCode/ko.md]: never mind about this, please disregard

Comment (EverTokki) [reply wiki/BBCode/ko.md]: no worriez

**Comment by MegaApplePi** [comment]
@EverTokki are you done with this?

**Comment by EverTokki** [comment]
@MegaApplePi I'll take a final look over it this weekend (Saturday), and will follow up.

**Comment by EverTokki** [comment]
@MegaApplePi Think it's good enough for a merge. Thank you.

**Pull request merged**